avalon_listener/Dockerfile

```diff
@@ -0,0 +1,196 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+# Description:
+#   Builds the environment needed to build Trusted Compute Framework.
+#
+#  Configuration (build) parameters
+#  - proxy configuration: https_proxy http_proxy ftp_proxy
+#  - sgx mode:
+#
+# Build:
+#   $ docker build docker -f docker/Dockerfile.tcf-dev -t tcf-dev
+#   if behind a proxy, you might want to add also below options
+#   --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy --build-arg ftp_proxy=$ftp_proxy
+#
+# Run:
+#   $ cd TrustedComputeFramework
+#   $ docker run -v $(pwd):/project/TrustedComputeFramework -it tcf-dev /bin/bash
+#   (to run with SGX HW, add options '--device=/dev/isgx -v /var/run/aesmd:/var/run/aesmd ')
+#   make clean && make
+
+# -------------=== listener build ===-------------
+
+FROM ubuntu:bionic
+
+# Ignore timezone prompt in apt
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Add necessary packages
+RUN apt-get update \
+ && apt-get install -y -q \
+    build-essential \
+    ca-certificates \
+    cmake \
+    git \
+    make \
+    pkg-config \
+    cmake \
+    libprotobuf-dev \
+    python3-dev \
+    python3-pip \
+    python3-toml \
+    python3-requests \
+    python3-colorlog \
+    python3-twisted \
+    swig \

```

Comment:
Are we using swig in listener? If not, please remove. Wold be good to remove unused python packages for avalon listener.

Reply:
ok